core/store/models/eth.go

```diff
@@ -216,6 +223,32 @@ func (f *FunctionSelector) UnmarshalJSON(input []byte) error {
 	return nil
 }
 
+// MarshalJSON returns the JSON encoding of f
+func (f FunctionSelector) MarshalJSON() ([]byte, error) {
+	return json.Marshal([]byte(f.String()))
+}
+
+// Value returns this instance serialized for database storage
+func (f FunctionSelector) Value() (driver.Value, error) {
+	return f.Bytes(), nil
+}
+
+// Scan returns the selector from its serialization in the database
+func (f FunctionSelector) Scan(value interface{}) error {
+	fmt.Println("value being deserialized", value)
```

Comment:
Maybe the Println(..)'s were left in by mistake?

Reply:
Yep. Fixed in bcca213d8bcab63efd5daf44652dac83517d332b